src/test/resources/features/DeleteCard.feature

```diff
@@ -2,7 +2,7 @@
 Feature: Delete card of a list
 
   Background: Create card to board in Trello
-    Given I Log in with user "{credentials.owner1}"
+    Given I Log in with user "{credentials.owner2}"
```

Comment:
Don't change this!

Reply:
I had to change the owner because I had some issues running the tests with owner1